Yoast/javascript
# FRO-209 table component

## Summary


This PR can be summarized in the following changelog entry:

* Introduce the new table component

## Relevant technical choices:

* N/A

## Test instructions

This PR can be tested by following these steps:

* Check out this PR
* Check if the standard table components are okay and inline with this design https://www.sketch.com/s/1ede3372-d1cc-4803-a4af-a06ca86aa0e3/a/ZAl4Yv#Inspector

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/FRO-209
